python/CMakeLists.txt

```diff
@@ -53,6 +54,23 @@ declare_mlir_python_extension(TorchMLIRPythonExtensions.Main
   EMBED_CAPI_LINK_LIBS
     TorchMLIRCAPI
   PRIVATE_LINK_LIBS
+    MLIRAffineToStandard
```

Comment:
I am somewhat surprised by having this list of C++ deps here... reading more and seeing if I can have a recommendation.

As written, this is linking part of MLIR into the Python extension itself and part into the CAPI .so... that is probably not what we want.